tests/run/test_unicode_string_tests.pxi

```diff
@@ -1295,9 +1295,9 @@ class MixinStrUnicodeUserStringTest:
             'this is the partition method', 'partition', 'ti')
 
         # from raymond's original specification
-        S = 'http://www.python.org'
+        S = 'https://www.python.org/'
```

Comment:
This is actually a part of a test, and it's mostly copied from CPython's test suite. We shouldn't change these strings.